packages/schema/src/dialects/oracledb.ts

```diff
@@ -308,6 +311,10 @@ export default class oracleDB implements SchemaInspector {
             AND "ct"."CONSTRAINT_PRIORITY" = 1
           LEFT JOIN "uc" "fk"
             ON "ct"."R_CONSTRAINT_NAME" = "fk"."CONSTRAINT_NAME"
+		  LEFT JOIN user_ind_columns aic
+			ON aic.TABLE_NAME="c"."TABLE_NAME" AND aic.COLUMN_NAME = "c"."COLUMN_NAME"
+		  LEFT JOIN user_indexes ai
+			ON aic.INDEX_NAME = ai.INDEX_NAME AND ai.CONSTRAINT_INDEX = 'NO' AND ai.UNIQUENESS = 'NONUNIQUE'
```

Comment:
`user_indexes` doesnt have filter by column, so had to use intermediate table `user_ind_columns`.

Reply:
also this is not composite index safe, to make it composite index same, we'd have to partition by index_name(as its same for multiple columns), since directus doesn't support composite indexes anyway, i went ahead with this to save a tiny bit of performance.